src/coreclr/jit/compiler.cpp

```diff
@@ -2284,6 +2284,11 @@ void Compiler::compSetProcessor()
 #ifdef TARGET_XARCH
     if (!compIsForInlining())
     {
+        if (canUseEvexEncoding())
+        {
+            codeGen->GetEmitter()->SetUseEVEXEncoding(true);
+            // TODO-XArch-AVX512: Revisit other flags to be set once avx512 instructions are added.
```

Comment:
This is referring to something like `codeGen->GetEmitter()->SetContainsAVX512(false)` and `codeGen->GetEmitter()->SetContains512bitAVX(false);`, correct?

Reply:
Yes. The logic to set these seem a little convoluted. We set them using SIMD size during lsra at multiple points. Would you prefer I add these as part of this commit?

Reply:
I don't think its necessary provided we consistently use `TODO-XArch-AVX512` so we don't risk missing such changes in later PRs.

Reply:
Okay. So, do I leave the correct comment as is or would you rather I expand the comment to directly mention the missing calls. Something like below?


```
if (canUseEvexEncoding())
{
            codeGen->GetEmitter()->SetUseEVEXEncoding(true);
            // TODO-XArch-AVX512: Revisit the following once avx512 instructions are added.
            //  codeGen->GetEmitter()->SetContainsAvx512(false);
            //  codeGen->GetEmitter()->SetContains512bitAvx(false);
}
```

Reply:
I think its fine as is for now. Was just wanting to confirm that's what was being considered.